### ISSUE TITLE
Can't login with Mac free trial account

### ISSUE THREAD

**Issue by marzent** [tyrone-sudeium/LaunchXIV#6]
Logging in via the native launcher works but using LaunchXIV returns an error.

I did some local sniffing and specifically the request towards `https://patch-gamever.ffxiv.com/http/win32/ffxivneo_release_game/{game_ver}/{session_id}` fails with
```
HTTP/1.1 401 Unauthorized
Server: nginx
Date: Mon, 01 Nov 2021 11:46:44 GMT
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive

0

```

**Comment by tyrone-sudeium** [comment]
~~Possibly related to #5 -- does your free trial account have one time password enabled on it (is it even possible to)?~~

I know that logins without one time passwords don't work well right now and it's very hard to work out why since the `patch-gamever.ffxiv.com` HTTP request doesn't appear in my HTTP proxy software when using the default launcher so I'm not sure what the default launcher is doing differently to me

EDIT: ok not related to one time passwords. Added one to my own free trial burner account and it's still a no go. Not sure what to do about this one until I can find a way to debug the default launcher

**Comment by tyrone-sudeium** [comment]
Should be fixed in [v0.3.3](/tyrone-sudeium/LaunchXIV/releases/tag/v0.3.3).
